src/lib/layers/editable-geojson-layer.js

```diff
@@ -171,11 +176,14 @@ export default class EditableGeoJsonLayer extends EditableLayer {
       drawFeature = this.getDrawFeature(selectedFeature, props.mode, null);
     }
 
+    const hintPoint = selectedFeatures.length ? this.state.hintPoint : [];
```

Comment:
I think we should stick with the term "intermediate edit handle" rather than "hint point".